charts/pwa/templates/deployment_cache.yaml

```diff
@@ -42,23 +47,22 @@ spec:
           - name: UPSTREAM_PWA
             value: "http://{{ include "pwa-main.fullname" . }}:{{ .Values.service.port }}"
           - name: ICM_BASE_URL
-            value: "{{ coalesce .Values.upstream.icm .Values.upstream.icmBaseURL }}"
+            value: "{{ .Values.upstream.icmBaseURL }}"
+        {{- if .Values.cache.multiChannel }}
```

Comment:
I still disagree to make this configuration optional. This is a deployment setting reflecting how NGInx is mapping external URLs to Application Server URLs. Making this part of the artifact being deployed sounds silly to me.

Reply:
Yes. With this change the implementation partner can unset the multi channel config of `values.yaml` (i.e. Helm) giving precedence to whatever contents of `nginx/multi-channel.yaml` (i.e. Source Code). As a result fixing #126 

I know this is open to debate. I also know that expectations from the implementation partner differ. They see the `multi-channel.yaml` as part of the Intershop PWA source code. So they use it to configure their URLs.

Reply:
Well I would say the change completely supports the way multi channel configuration should be done in your opinion while adding the also requested and used way of configuring it via project source code.
So there is no need for debate, the customer and partners can still use the "right" way. ;)